source/declarators.tex

```diff
@@ -2110,7 +2110,7 @@
     attribute-specifier-seq\opt decl-specifier-seq\opt declarator virt-specifier-seq\opt{} \terminal{ = default ;}
 \end{ncbnf}
 
-is called an \grammarterm{explicitly-defaulted} definition.
+is called an \term{explicitly-defaulted} definition.
 A function that is explicitly defaulted shall
```

Comment:
This should be `\defn{explicitly-defaulted definition}`, probably with "definition!explicitly-defaulted" as the main index entry and a "see" pointer for "explicitly-defaulted definition".